pyproject.toml

```
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "dictionaryutils"
-version = "3.4.8"
+version = "3.4.9"
 description = "Python wrapper and metaschema for datadictionary."
 authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/uc-cdis/dictionaryutils"
 
 [tool.poetry.dependencies]
-python = ">=3.6,<3.10"
+python = "^3.9"
```

```suggestion
python = ">=3.9,<4"
```
we've run into issues with the python version before when we used `^` in libraries, this ^ is usually flexible enough